File: 프로그래머스/lv0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.py
def solution(n):
    answer = 0
    
    a = n / 7
    b = n % 7
    
    if b == 0:
        answer = a * 1
    else:
        answer = a * 1 + 1
        
    answer = int(answer)
    print(answer)

    return answer

File: 프로그래머스/lv0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.js
function solution(n) {
    var answer = 0;
    
    a = Math.floor(n / 7)
    b = n % 7
    
    if (b === 0) {
        answer = a * 1
    } else {
        answer = a * 1 + 1
    }
    
    return answer;
}